yarn.lock

```diff
@@ -3793,6 +3793,16 @@ amdefine@>=0.0.4:
   resolved "https://registry.yarnpkg.com/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
+amplify-frontend-javascript@2.13.2:
```

Comment:
Unrelated change to this PR, please remove it.

Reply:
removed